Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -200,667 +144,69 @@ class EditorBatchedTest(EditorSharedTest):
-        test owns it.
-        :workspace: The workspace object in case an AssetProcessor object needs to be created
-        :editor_test_data: The test data from calling editor_test_data()
-        :return: None
-        """
-        try:
-            # Start-up an asset processor if we are not already managing one
-            if editor_test_data.asset_processor is None:
-                if not process_utils.process_exists("AssetProcessor", ignore_extensions=True):
-                    editor_utils.kill_all_ly_processes(include_asset_processor=True)
-                    editor_test_data.asset_processor = AssetProcessor(workspace)
-                    editor_test_data.asset_processor.start()
-                else:  # If another AP process already exists, do not kill it as we do not manage it
-                    editor_utils.kill_all_ly_processes(include_asset_processor=False)
-            else:  # Make sure existing asset processor wasn't closed by accident
-                editor_test_data.asset_processor.start()
-        except Exception as ex:
-            editor_test_data.asset_processor = None
-            raise ex
-
-    def _setup_editor_test(self, editor: ly_test_tools.launchers.platforms.base.Launcher,
-                           workspace: ly_test_tools._internal.managers.workspace.AbstractWorkspaceManager,
-                           editor_test_data: TestData) -> None:
-        """
-        Sets up an editor test by preparing the Asset Processor, killing all other O3DE processes, and configuring
-        :editor: The launcher Editor object
-        :workspace: The test Workspace object
-        :editor_test_data: The TestData from calling editor_test_data()
-        :return: None
-        """
-        self._prepare_asset_processor(workspace, editor_test_data)
-        editor_utils.kill_all_ly_processes(include_asset_processor=False)
-        editor.configure_settings()
-
-    @staticmethod
-    def _get_results_using_output(test_spec_list: list[EditorTest], output: str, editor_log_content: str) -> dict[str, Result.ResultType]:
-        """
-        Utility function for parsing the output information from the editor. It deserializes the JSON content printed in
-        the output for every test and returns that information.
-        :test_spec_list: The list of EditorTests
-        :output: The Editor from Editor.get_output()
-        :editor_log_content: The contents of the editor log as a string
-        :return: A dict of the tests and their respective Result objects
+    # Maximum time (seconds) for waiting for a crash file to finish being dumped to disk
+    _TIMEOUT_CRASH_LOG = 20
+    # Return code for test failure
+    _TEST_FAIL_RETCODE = 0xF
+    # Test class to use for single test collection
+    single_test_class = EditorSingleTest
+    # Test class to use for shared test collection
+    shared_test_class = EditorSharedTest
```

Comment:
These don't appear to be variables intended for user modification, should add a leading underscore